dj_hetmech_app/views.py

```diff
@@ -34,9 +34,6 @@ class NodeViewSet(ReadOnlyModelViewSet):
     http_method_names = ['get']
     serializer_class = NodeSerializer
     filter_backends = (filters.SearchFilter, )
-    # See the following page for "search" implementation and other filter options:
-    # https://www.django-rest-framework.org/api-guide/filtering/
-    search_fields = ('identifier', 'metanode__identifier', 'name')
```

Comment:
I think we want to keep the ability to have an exact match on `identifier`. Possibly even a case insensitive exact match. We probably should give the exact identifier match the top score possible.